examples/account_management/link_accounts.rb

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env ruby
+# Encoding: utf-8
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This example demonstrates how to link two already existing accounts.
+
+require 'optparse'
+require 'google/ads/google_ads'
+
+def link_accounts(manager_customer_id, client_customer_id)
+  # GoogleAdsClient will read a config file from
+  # ENV['HOME']/google_ads_config.rb when called without parameters
+  client = Google::Ads::GoogleAds::GoogleAdsClient.new
+
+  # This example assumes that the same credentials will work for both accounts,
+  # but that may not be the case. If you need to use different credentials
+  # for each account, then you may either update the client configuration or
+  # instantiate two clients, one for each set of credentials. Always make sure
+  # to update the configuration before fetching any services you need to use.
+
+  # Extend an invitation while authenticating as the manager account.
+  client.configure do |config|
+    config.login_customer_id = manager_customer_id.to_i
+  end
+
+  client_link_service = client.service(:CustomerClientLink)
+
+  client_link = client.resource(:CustomerClientLink)
+  client_link.client_customer = client.wrapper.string(
+    client.path.customer(client_customer_id),
+  )
+  client_link.status = :PENDING
+
+  client_link_operation = client.operation(:CustomerClientLink)
+  client_link_operation['create'] = client_link
+
+  response = client_link_service.mutate_customer_client_link(
+    manager_customer_id,
+    client_link_operation,
+  )
+
+  client_link_resource_name = response.result.resource_name
+  puts "Extended an invitation from account #{manager_customer_id} to account " \
+      "#{client_customer_id} with resource name #{client_link_resource_name}."
```

Comment:
nit: "client link resource name"
As written, could be misinterpreted as the customer resource name